github.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+CIRCLE_BRANCH=pr-tesr
```

Shellcheck

[warning] CIRCLE_BRANCH appears unused. Verify use (or export if used externally).

(SC2034)